src/WaterPokemon.java

```diff
@@ -0,0 +1,127 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon {
+    private final List<String> attacks = Arrays.asList("surf", "hydroPump", "hydroCanon", "rainDance");
+
+    public WaterPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound);
+    }
+
+    @Override
+    public String getType() {
+        return "water";
+    }
+
+
+    public void surf(Pokemon pokemon, Pokemon enemy) {
+        System.out.println(pokemon.getName() + " attacks " + enemy.getName() + " with surf");
+        switch (enemy.getType()) {
```

Comment:
Deze switch statement is voor elke aanvalsmethode (bijna) hetzelfde. Je zou eens kunnen kijken of je daar een helper methode van kunt maken. Dat maakt je code vaak een stuk leesbaarder.